saleor/order/utils.py

```diff
@@ -343,10 +343,50 @@ def match_orders_with_new_user(user: User) -> None:
     Order.objects.confirmed().filter(user_email=user.email, user=None).update(user=user)
 
 
+def chunk_products(products, product_limit):
+    """Split products to list of chunks.
+
+    Each chunk represents products per page, product_limit defines chunk size.
+    """
+    chunks = []
+    for i in range(0, len(products), product_limit):
+        limit = i + product_limit
+        chunks.append(products[i:limit])
+    return chunks
+
+
+def get_product_limit_first_page(products):
+    MAX_PRODUCTS_WITH_TABLE = 3
+    MAX_PRODUCTS_WITHOUT_TABLE = 4
+
+    if len(products) < MAX_PRODUCTS_WITHOUT_TABLE:
+        return MAX_PRODUCTS_WITH_TABLE
+
+    return MAX_PRODUCTS_WITHOUT_TABLE
+
+
 def generate_invoice_pdf_for_order(invoice):
-    logo_path = static_finders.find("images/logo-light.svg")
+    logo_path = static_finders.find("images/logo.svg")
```

Comment:
Shouldn't be a part of our basic Invoice plugin? Will Saleor use these functions somewhere else? If not I assume that we should move them to InvoicePlugin logic

Reply:
Can be moved, perhaps will do so, but in a separate PR when I will have the logic behind core plugin (generating invoice by Saleor)

Reply:
Okay, so maybe a gh issue to remember about it? No need to keep pdf logic in saleor core as we will use it only inside a plugin.